### PR TITLE
Adds ability to move tracks up and down within the Goldify Playlist table

### LINKED ISSUE #37: Ability to re-order songs in playlist
**Is your feature request related to a problem? Please describe.**

Users are currently not able to re-order playlists - they can only add and remove, and songs are kept in the order they were added
**Describe the solution you'd like**

Users should be able to pick the order of the songs in their playlist, regardless of when they were added
**Describe alternatives you've considered**

this is not a very high priority, so it's fine if we push this off for a bit

### PR DESCRIPTION
**Pull request by MattGrantUMich**
- Adds ability to move tracks up and down in Goldify playlist
- Triggers save/cancel buttons to appear on movement of the tracks
- Uses three bar icon (universal move up/down icon)

Closes #37

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ import GoldifyApp from "./js/GoldifyApp";
 import reportWebVitals from "./js/utils/reportWebVitals";
 
 ReactDOM.render(
-  <React.StrictMode>
+  <React.Fragment>
     <GoldifyApp />
-  </React.StrictMode>,
+  </React.Fragment>,
```
Comment (kyrod) [inline src/index.js:10]: Is this required for the sortable arrays?

File: src/js/solo/goldify-playlist/GoldifyPlaylistData.js [hunk 2]
```diff
@@ -59,19 +61,23 @@ class GoldifyPlaylistData extends Component {
...
     );
   }
 
-  setInitialGoldifyPlaylistData(playlistTrackData) {
+  setURIListFromPlaylistData(playlistData) {
     var uriList = [];
-    for (var i = 0; i < playlistTrackData.items.length; ++i) {
-      var track = playlistTrackData.items[i].track;
+    for (var i = 0; i < playlistData.length; ++i) {
+      var track = playlistData[i].track;
```
Comment (kyrod) [inline src/js/solo/goldify-playlist/GoldifyPlaylistData.js:73]: 👍

File: src/js/utils/GoldifyPlaylistDataElements.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  sortableContainer,
+  sortableElement,
+  sortableHandle,
+} from "react-sortable-hoc";
+
+export const DragHandle = sortableHandle(() => (
+  <span className="move-row-icon"></span>
+));
+
+export const SortableItem = sortableElement(
```
Comment (kyrod) [inline src/js/utils/GoldifyPlaylistDataElements.js:14]: Ooh this is nice

**Review by kyrod — APPROVED** [review]
Looks good! I like the removal of the .items key, should be much easier to work with this object now

File: src/index.js (hunk 1 above)
Comment (MattGrantUMich) [reply src/index.js:10]: I forgot to mention this. We should make an issue for this one, but there's PR waiting on the end of the people who made the new component I pulled in which would remove issues that are brought about (in console only) with StrictMode but not with Fragment.

**Pull request merged**